Extensions/ContentTypes/Migrations.cs

```diff
@@ -0,0 +1,42 @@
+﻿using OrchardCore.Autoroute.Models;
+using OrchardCore.ContentManagement.Metadata;
+using OrchardCore.ContentManagement.Metadata.Settings;
+using OrchardCore.Data.Migration;
+using static Lombiq.HelpfulExtensions.Extensions.ContentTypes.ContentTypes;
+
+namespace Lombiq.HelpfulExtensions.Extensions.ContentTypes
+{
+    public class Migrations : DataMigration
+    {
+        IContentDefinitionManager _contentDefinitionManager;
+
+
+        public Migrations(IContentDefinitionManager contentDefinitionManager)
+        {
+            _contentDefinitionManager = contentDefinitionManager;
+        }
+
+
+        public int Create()
+        {
+            _contentDefinitionManager.AlterTypeDefinition(Page, builder => builder

```

Comment:
So this content type is not available anywhere else? Same for the widgets? If yes, let's document in the Readme why we're including them.

Reply:
Nope. Only in built-in recipes.